framework/src/actions/CheckOutputAction.C

```diff
@@ -50,10 +50,10 @@ CheckOutputAction::checkVariableOutput(const std::string & task)
   {
     // Loop through the actions for the given task
     const std::vector<Action *> & actions = _awh.getActionsByName(task);
-    for (std::vector<Action *>::const_iterator it = actions.begin(); it != actions.end(); ++it)
+    for (auto act : actions)
```

Comment:
I would expect zero change for naked pointers. However, I would expect a tiny increase for shared pointers if you copy them (two atomic operations).